webcompat/static/js/lib/bugform.js

```diff
@@ -457,11 +457,8 @@ function BugForm() {
         // remove the last embedded image URL
         // Note: this could fail in weird ways depending on how
         // the user has edited the descField.
-        this.descField.val(function(idx, value) {
-          return value.replace(
-            /\[!\[[^\]]+\]\([^\)]+\)\]\([^\.]+.(?:bmp|gif|jpe*g*)\)$/,
-            ""
-          );
+        this.stepsToReproduceField.val(function(idx, value) {
+          return value.replace(/\[!\[[^\]]+\]\([^\)]+\)\]\([^\)]+\)$/);
```

Comment:
Apart of this. Testing in the Web Console of a browser with the correct syntax.

```js
> var image = "[![Screenshot Description](http://localhost:5000/uploads/2017/7/14afaf39-a749-4a55-8fc1-5b3bb7f040e7-thumb.jpg)](http://localhost:5000/uploads/2017/7/14afaf39-a749-4a55-8fc1-5b3bb7f040e7.jpg)" 
undefined 
> image.replace(/\[!\[[^\]]+\]\([^\)]+\)\]\([^\)]+\)$/, '') 
""
```

This is working.